message.go

```diff
@@ -330,6 +335,7 @@ func (msg *Message) IsValid() error {
 			return InvalidMessageError("missing signature")
 		}
 	}
+
```

Comment:
Looks like a spurious empty line.

Reply:
Regarding this -- once #301 and #303 are merged, I will add more linters to CI, including those that will find things like this (and can also be used locally before sending a PR). Talking about golangci-lint and friends.

Reply:
sounds good @kolyshkin, adding linting was also on my bucket list